docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,336 @@
+:Number:BP000
+:Title: Blueprint specification and requirements
+:Author: Beraldo Leal <bleal@redhat.com>
+:Reviewers: Cleber Rosa <crosa@redhat.com>, Jan Richter <jarichte@redhat.com>,
+            Plamen Dimitrov <pdimitrov@pevogam.com>, Willian Rampazzo
+            <willianr@redhat.com>
+:Created: 29-Sep-2020
+:Type: Meta Blueprint
+:Status: Approved
+
+.. contents:: Table of Contents
+
+TL;DR
+*****
+
+Having better records of architectural decisions in our repository is a good
+way to socialize our technical decisions, improve the code review process,
+avoid unnecessary code, and balance the workload among other developers. We are
+already using the “blueprint” documents for a while and still learning how to
+write them. Although we have a basic idea of what is a blueprint, some may not
+understand the concept because we are missing one meta-document that describes
+the blueprint's basic notions. This document is a blueprint to specify how we
+should write blueprints from now on.
+
+Motivation
+**********
+
+Depending on the project size, having very well defined and structured
+documents about the architecture decisions seems like an overkill, but could
+help projects of any size, including Avocado, save time, make better decisions
```

Comment:
I generally think Avocado is far from a small project so definitely useful to have the blueprints here. They probably also arose naturally from the complexity and impactful decisions involved in each architectural change.
(just a small comment, not requesting to change anything)